### ISSUE TITLE
`LOV_MAX_STRIPE_COUNT` undefined

### ISSUE THREAD

**Issue by eschnett** [open-mpi/ompi#802]
I am building OpenMPI v2.x-dev-187-g8516895. I now see
```
/home/eschnett/datura/software/src/openmpi-v2.x-dev-187-g8516895/src/openmpi-v2.x-dev-187-g8516895/ompi/mca/fs/lustre/fs_lustre_file_open.c:48:5: error: use of undeclared identifier 'LOV_MAX_STRIPE_COUNT'
```
on several systems (e.g. Hopper at NERSC). These errors were not present in earlier snapshots.

**Comment by edgargabriel** [comment]
I commit earlier today a patch for that 55f0e1a1f859a0f16c4de3b1edf5ed4ba13e4818 which should take care of that, if you have a chance could you please let me know whether it works?

**Comment by eschnett** [comment]
I left a comment for the commit.

**Comment by edgargabriel** [comment]
fixed with commits 55f0e1a1f859a0f16c4de3b1edf5ed4ba13e4818 and 9f369ba515417a7c3ce5d634c218918546789dba
